loc/translations-export/vscode-powerplatform.xlf

```diff
@@ -811,6 +817,9 @@ The second line should be '[TRANSLATION HERE](command:powerplatform-walkthrough.
     <trans-unit id="pacCLI.pacSolutionHelp.title">
       <source xml:lang="en">Show PAC Solution Help</source>
     </trans-unit>
+    <trans-unit id="microsoft.powerplatform.pages.actionsHub.activeSite.uploadSite.title">
+      <source xml:lang="en">Upload changes</source>
```

Comment:
@priyanshu92 Should this be "Upload changes using PAC CLI" instead?

Reply:
@ashishchoudhary001 Since we are now opening the terminal instead of running the download in background, we are keeping this as is. The terminal will give additional context that the operation is happening via PAC CLI.